04-26 18:17:30 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "CREATE TABLE ORDER[*](
ID INT PRIMARY KEY,
COUNTRY VARCHAR(50),
ITEMTOTAL DECIMAL,
TOTAL DECIMAL)"; se esperaba "identifier"
Syntax error in SQL statement "CREATE TABLE ORDER[*](
ID INT PRIMARY KEY,
COUNTRY VARCHAR(50),
ITEMTOTAL DECIMAL,
TOTAL DECIMAL)"; expected "identifier"; SQL statement:
CREATE TABLE ORDER(
ID INT PRIMARY KEY,
COUNTRY VARCHAR(50),
ITEMTOTAL DECIMAL,
TOTAL DECIMAL) [42001-154]
04-26 18:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "CREATE TABLE ORDER[*](
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50),
ITEMTOTAL DECIMAL(8,2),
TOTAL DECIMAL(8,2))"; se esperaba "identifier"
Syntax error in SQL statement "CREATE TABLE ORDER[*](
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50),
ITEMTOTAL DECIMAL(8,2),
TOTAL DECIMAL(8,2))"; expected "identifier"; SQL statement:
CREATE TABLE ORDER(
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50),
ITEMTOTAL DECIMAL(8,2),
TOTAL DECIMAL(8,2)) [42001-154]
04-26 18:24:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "CREATE TABLE ORDER[*](
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50))"; se esperaba "identifier"
Syntax error in SQL statement "CREATE TABLE ORDER[*](
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50))"; expected "identifier"; SQL statement:
CREATE TABLE ORDER(
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50)) [42001-154]
04-26 18:24:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "CREATE TABLE 'ORDER'[*](
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50))"; se esperaba "identifier"
Syntax error in SQL statement "CREATE TABLE 'ORDER'[*](
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50))"; expected "identifier"; SQL statement:
CREATE TABLE 'ORDER'(
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50)) [42001-154]
04-26 18:24:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "CREATE TABLE [[*]ORDER](
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50)) "; se esperaba "identifier"
Syntax error in SQL statement "CREATE TABLE [[*]ORDER](
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50)) "; expected "identifier"; SQL statement:
CREATE TABLE [order](
ID INT PRIMARY KEY,
COUNTRY  VARCHAR(50)) [42001-154]
04-26 18:36:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabla "TABLE" no encontrada
Table "TABLE" not found; SQL statement:
delete table probando [42102-154]
04-26 18:36:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabla "TABLE" no encontrada
Table "TABLE" not found; SQL statement:
delete table PROBANDO [42102-154]
04-26 18:37:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabla "PROBANDO" no encontrada
Table "PROBANDO" not found; SQL statement:
delete PROBANDO [42102-154]
04-26 18:37:32 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Tabla "PROBANDO" no encontrada
Table "PROBANDO" not found; SQL statement:
DROP TABLE PROBANDO [42102-154]
04-26 18:40:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "DROP TABLE ORDER[*] "; se esperaba "identifier"
Syntax error in SQL statement "DROP TABLE ORDER[*] "; expected "identifier"; SQL statement:
drop table order [42001-154]
04-26 18:40:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "DROP TABLE ORDER[*] "; se esperaba "identifier"
Syntax error in SQL statement "DROP TABLE ORDER[*] "; expected "identifier"; SQL statement:
drop table order [42001-154]
04-26 18:40:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Tabla "order" no encontrada
Table "order" not found; SQL statement:
drop table "order" [42102-154]
04-26 18:40:52 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Tabla "rdOer" no encontrada
Table "rdOer" not found; SQL statement:
drop table "rdOer" [42102-154]
04-26 18:41:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "CREATE TABLE ""Order""(
ID INT PRIMARY_KEY[*],
COUNTRY VARCHAR(50),
ITEMTOTAL DECIMAL(10,2),
TOTAL DECIMAL (10,2)
) "; se esperaba ")"
Syntax error in SQL statement "CREATE TABLE ""Order""(
ID INT PRIMARY_KEY[*],
COUNTRY VARCHAR(50),
ITEMTOTAL DECIMAL(10,2),
TOTAL DECIMAL (10,2)
) "; expected ")"; SQL statement:
create table "Order"(
Id int primary_key,
Country varchar(50),
ItemTotal decimal(10,2),
Total decimal (10,2)
) [42001-154]
04-26 18:41:46 jdbc[4]: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-154]
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:98)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:642)
	at org.h2.server.web.WebApp.login(WebApp.java:892)
	at org.h2.server.web.WebApp.process(WebApp.java:205)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:164)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
04-26 18:55:07 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Tabla "order" no encontrada
Table "order" not found; SQL statement:
drop table "order" [42102-154]
04-26 18:56:35 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Tabla "Order" no encontrada
Table "Order" not found; SQL statement:
drop table "Order" [42102-154]
04-26 19:05:33 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Columna "ESP" no encontrada
Column "ESP" not found; SQL statement:
insert into Shipping values("ESP",12) [42122-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:134)
	at org.h2.command.dml.Insert.prepare(Insert.java:233)
	at org.h2.command.Parser.prepare(Parser.java:202)
	at org.h2.command.Parser.prepareCommand(Parser.java:214)
	at org.h2.engine.Session.prepareLocal(Session.java:426)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1100)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1308)
	at org.h2.server.web.WebApp.query(WebApp.java:994)
	at org.h2.server.web.WebApp$1.next(WebApp.java:957)
	at org.h2.server.web.WebApp$1.next(WebApp.java:960)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
04-26 19:07:09 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "INSERT INTO SHIPPING VALUES ('PER',120 "; se esperaba ")"
Syntax error in SQL statement "INSERT INTO SHIPPING VALUES ('PER',120 "; expected ")"; SQL statement:
insert into Shipping values ('PER',120 [42001-154]
04-26 23:39:36 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Columna "PER" no encontrada
Column "PER" not found; SQL statement:
select Cost from Shipping where Country="PER" [42122-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:134)
	at org.h2.expression.Comparison.optimize(Comparison.java:156)
	at org.h2.command.dml.Select.prepare(Select.java:797)
	at org.h2.command.Parser.prepare(Parser.java:202)
	at org.h2.command.Parser.prepareCommand(Parser.java:214)
	at org.h2.engine.Session.prepareLocal(Session.java:426)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1100)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1308)
	at org.h2.server.web.WebApp.query(WebApp.java:994)
	at org.h2.server.web.WebApp$1.next(WebApp.java:957)
	at org.h2.server.web.WebApp$1.next(WebApp.java:960)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
04-26 23:45:02 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "SELECT ID, COUNTRY, ITEMTOTAL, TOTAL FROM 'Order'[*] WHERE ID=? "; se esperaba "identifier"
Syntax error in SQL statement "SELECT ID, COUNTRY, ITEMTOTAL, TOTAL FROM 'Order'[*] WHERE ID=? "; expected "identifier"; SQL statement:
select Id, Country, ItemTotal, Total from 'Order' where Id=? [42001-154]
04-26 23:50:31 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Columna "COUNTRY" no encontrada
Column "COUNTRY" not found; SQL statement:
select Id, Country, ItemTotal, Total from "ORDER" where Id=? [42122-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:134)
	at org.h2.command.dml.Select.prepare(Select.java:794)
	at org.h2.command.Parser.prepare(Parser.java:202)
	at org.h2.command.Parser.prepareCommand(Parser.java:214)
	at org.h2.engine.Session.prepareLocal(Session.java:426)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1100)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:71)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:243)
	at examples.coupleddesign.DataAccess.getOrder(DataAccess.java:34)
	at examples.coupleddesign.OrderServices.getOrder(OrderServices.java:20)
	at examples.coupleddesign.OrderServicesTests.Save_ValidOrder_TheOrderIsPersisted(OrderServicesTests.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.runners.BlockJUnit4ClassRunner.runNotIgnored(BlockJUnit4ClassRunner.java:79)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:71)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:49)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
04-27 00:10:48 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "SELECT * FROM ORDER[*] "; se esperaba "identifier"
Syntax error in SQL statement "SELECT * FROM ORDER[*] "; expected "identifier"; SQL statement:
select * from order [42001-154]
04-27 00:10:54 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "SELECT * FROM ORDER[*] "; se esperaba "identifier"
Syntax error in SQL statement "SELECT * FROM ORDER[*] "; expected "identifier"; SQL statement:
select * from order [42001-154]
04-27 00:10:59 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Tabla "order" no encontrada
Table "order" not found; SQL statement:
select * from "order" [42102-154]
04-27 00:14:28 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "CREATE TABLE ""ORDER""(
ID INT PRIMARY_KEY[*],
COUNTRY VARCHAR(50),
ITEMTOTAL DECIMAL(10,2),
TOTAL DECIMAL(10,2) ) "; se esperaba ")"
Syntax error in SQL statement "CREATE TABLE ""ORDER""(
ID INT PRIMARY_KEY[*],
COUNTRY VARCHAR(50),
ITEMTOTAL DECIMAL(10,2),
TOTAL DECIMAL(10,2) ) "; expected ")"; SQL statement:
create table "ORDER"(
id int primary_key,
country varchar(50),
itemtotal decimal(10,2),
total decimal(10,2) ) [42001-154]
04-27 00:30:15 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "INSERT INTO 'Order'[*] VALUES(?, ?, ?, ?) "; se esperaba "identifier"
Syntax error in SQL statement "INSERT INTO 'Order'[*] VALUES(?, ?, ?, ?) "; expected "identifier"; SQL statement:
insert into 'Order' values(?, ?, ?, ?) [42001-154]
05-10 20:13:39 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "SELEXT[*] * FROM SHIPPING "; se esperaba "SELECT, SET, SAVEPOINT, SCRIPT, SHUTDOWN, SHOW, {"
Syntax error in SQL statement "SELEXT[*] * FROM SHIPPING "; expected "SELECT, SET, SAVEPOINT, SCRIPT, SHUTDOWN, SHOW, {"; SQL statement:
selext * from shipping [42001-154]
05-10 23:27:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "DROP EMPLOYEE[*] "; se esperaba "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE"
Syntax error in SQL statement "DROP EMPLOYEE[*] "; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE"; SQL statement:
drop employee [42001-154]
05-11 00:11:53 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "DROP TABLE ORDER[*] "; se esperaba "identifier"
Syntax error in SQL statement "DROP TABLE ORDER[*] "; expected "identifier"; SQL statement:
drop table order [42001-154]
05-11 00:12:37 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Tabla "EMPLOYEE" no encontrada
Table "EMPLOYEE" not found; SQL statement:
insert into employee('angel','salazar',2010/12/30) [42102-154]
05-11 00:12:42 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Tabla "EMPLOYEEA" no encontrada
Table "EMPLOYEEA" not found; SQL statement:
insert into employeea('angel','salazar',2010/12/30) [42102-154]
05-11 00:12:43 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "INSERT INTO EMPLOYEES('angel'[*],'salazar',2010/12/30) "; se esperaba "identifier"
Syntax error in SQL statement "INSERT INTO EMPLOYEES('angel'[*],'salazar',2010/12/30) "; expected "identifier"; SQL statement:
insert into employees('angel','salazar',2010/12/30) [42001-154]
05-11 00:13:15 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Imposible interpretar la constante de fecha "5", causa: "format yyyy-mm-dd"
Cannot parse date constant "5", cause: "format yyyy-mm-dd"; SQL statement:
insert into employees(firstname,lastname,hiredate) values('angel','salazar',2010/12/30) -- ('angel', 'salazar', 5) [90009-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.util.DateTimeUtils.parseDateTime(DateTimeUtils.java:259)
	at org.h2.value.ValueDate.parseDate(ValueDate.java:45)
	at org.h2.value.Value.convertTo(Value.java:812)
	at org.h2.table.Column.convert(Column.java:151)
	at org.h2.command.dml.Insert.insertRows(Insert.java:114)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:212)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1308)
	at org.h2.server.web.WebApp.query(WebApp.java:994)
	at org.h2.server.web.WebApp$1.next(WebApp.java:957)
	at org.h2.server.web.WebApp$1.next(WebApp.java:960)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
05-11 00:13:32 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Imposible interpretar la constante de fecha "1968", causa: "format yyyy-mm-dd"
Cannot parse date constant "1968", cause: "format yyyy-mm-dd"; SQL statement:
insert into employees(firstname,lastname,hiredate) values('angel','salazar',2010-12-30) -- ('angel', 'salazar', 1968) [90009-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.util.DateTimeUtils.parseDateTime(DateTimeUtils.java:259)
	at org.h2.value.ValueDate.parseDate(ValueDate.java:45)
	at org.h2.value.Value.convertTo(Value.java:812)
	at org.h2.table.Column.convert(Column.java:151)
	at org.h2.command.dml.Insert.insertRows(Insert.java:114)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:212)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1308)
	at org.h2.server.web.WebApp.query(WebApp.java:994)
	at org.h2.server.web.WebApp$1.next(WebApp.java:957)
	at org.h2.server.web.WebApp$1.next(WebApp.java:960)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
05-11 00:14:03 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Tabla "EMPLOYEE" no encontrada
Table "EMPLOYEE" not found; SQL statement:
select * from employee [42102-154]
05-11 03:31:09 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Columna "PACHECO" no encontrada
Column "PACHECO" not found; SQL statement:
select Id, FirstName, LastName, HireDate from Employees where LastName=Pacheco [42122-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:134)
	at org.h2.expression.Comparison.optimize(Comparison.java:156)
	at org.h2.command.dml.Select.prepare(Select.java:797)
	at org.h2.command.Parser.prepare(Parser.java:202)
	at org.h2.command.Parser.prepareCommand(Parser.java:214)
	at org.h2.engine.Session.prepareLocal(Session.java:426)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1100)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:71)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:243)
	at org.apache.commons.dbcp.DelegatingConnection.prepareStatement(DelegatingConnection.java:281)
	at org.apache.commons.dbcp.PoolingDataSource$PoolGuardConnectionWrapper.prepareStatement(PoolingDataSource.java:313)
	at examples.databasetesting.EmployeeJDBC.find(EmployeeJDBC.java:32)
	at examples.databasetesting.EmployeeJDBCTest.find_withLastNameFilter_returnTheEmployeesWithTheExactLastName(EmployeeJDBCTest.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:82)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:72)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:240)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:180)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:49)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
05-11 03:31:09 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Imposible interpretar la constante de fecha "1969", causa: "format yyyy-mm-dd"
Cannot parse date constant "1969", cause: "format yyyy-mm-dd"; SQL statement:
select Id, FirstName, LastName, HireDate from Employees where HireDate between 2010-12-29 and 2010-12-30 [90009-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.util.DateTimeUtils.parseDateTime(DateTimeUtils.java:259)
	at org.h2.value.ValueDate.parseDate(ValueDate.java:45)
	at org.h2.value.Value.convertTo(Value.java:812)
	at org.h2.expression.Comparison.getValue(Comparison.java:229)
	at org.h2.expression.ConditionAndOr.getValue(ConditionAndOr.java:83)
	at org.h2.expression.Expression.getBooleanValue(Expression.java:180)
	at org.h2.command.dml.Select.queryFlat(Select.java:514)
	at org.h2.command.dml.Select.queryWithoutCache(Select.java:617)
	at org.h2.command.dml.Query.query(Query.java:290)
	at org.h2.command.dml.Query.query(Query.java:260)
	at org.h2.command.dml.Query.query(Query.java:37)
	at org.h2.command.CommandContainer.query(CommandContainer.java:78)
	at org.h2.command.Command.executeQuery(Command.java:181)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:96)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	at examples.databasetesting.EmployeeJDBC.find(EmployeeJDBC.java:34)
	at examples.databasetesting.EmployeeJDBCTest.find_withHireDateFilters_returnTheEmployeesBetweenHiredates(EmployeeJDBCTest.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:82)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:72)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:240)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:180)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:49)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
05-11 03:38:51 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "SELECT ID, FIRSTNAME, LASTNAME, HIREDATE FROM EMPLOYEES WHERE LASTNAME='Pacheco'HIREDATE[*] BETWEEN '2010-12-29' AND '2010-12-30' "
Syntax error in SQL statement "SELECT ID, FIRSTNAME, LASTNAME, HIREDATE FROM EMPLOYEES WHERE LASTNAME='Pacheco'HIREDATE[*] BETWEEN '2010-12-29' AND '2010-12-30' "; SQL statement:
select Id, FirstName, LastName, HireDate from Employees where LastName='Pacheco'HireDate between '2010-12-29' and '2010-12-30' [42000-154]
05-11 16:56:17 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Tabla "EMPLOYEE" no encontrada
Table "EMPLOYEE" not found; SQL statement:
insert into Employee values(?, ?, ?, ?) [42102-154]
05-11 16:56:17 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Tabla "EMPLOYEE" no encontrada
Table "EMPLOYEE" not found; SQL statement:
insert into Employee values(?, ?, ?, ?) [42102-154]
05-11 16:56:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Parametro "#1" no está fijado
Parameter "#1" is not set; SQL statement:
insert into Employees values(?, ?, ?, ?) [90012-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.expression.Parameter.checkSet(Parameter.java:73)
	at org.h2.command.Prepared.checkParameters(Prepared.java:162)
	at org.h2.command.CommandContainer.update(CommandContainer.java:68)
	at org.h2.command.Command.executeUpdate(Command.java:212)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at examples.databasetesting.EmployeeJDBC.create(EmployeeJDBC.java:82)
	at examples.databasetesting.EmployeeJDBCTest.create_theEmployeeIsPersisted(EmployeeJDBCTest.java:82)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:82)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:72)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:240)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:180)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:49)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
05-11 16:57:25 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: La cantidad de columnas no coincide
Column count does not match; SQL statement:
insert into Employees values(?, ?, ?) [21002-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.command.dml.Insert.prepare(Insert.java:228)
	at org.h2.command.Parser.prepare(Parser.java:202)
	at org.h2.command.Parser.prepareCommand(Parser.java:214)
	at org.h2.engine.Session.prepareLocal(Session.java:426)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1100)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:71)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:243)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:1028)
	at org.apache.commons.dbcp.DelegatingConnection.prepareStatement(DelegatingConnection.java:508)
	at org.apache.commons.dbcp.PoolingDataSource$PoolGuardConnectionWrapper.prepareStatement(PoolingDataSource.java:400)
	at examples.databasetesting.EmployeeJDBC.create(EmployeeJDBC.java:75)
	at examples.databasetesting.EmployeeJDBCTest.create_theEmployeeIsPersisted(EmployeeJDBCTest.java:82)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:82)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:72)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:240)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:180)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:49)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
05-11 16:57:25 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: La cantidad de columnas no coincide
Column count does not match; SQL statement:
insert into Employees values(?, ?, ?) [21002-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.command.dml.Insert.prepare(Insert.java:228)
	at org.h2.command.Parser.prepare(Parser.java:202)
	at org.h2.command.Parser.prepareCommand(Parser.java:214)
	at org.h2.engine.Session.prepareLocal(Session.java:426)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1100)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:71)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:243)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:1028)
	at org.apache.commons.dbcp.DelegatingConnection.prepareStatement(DelegatingConnection.java:508)
	at org.apache.commons.dbcp.PoolingDataSource$PoolGuardConnectionWrapper.prepareStatement(PoolingDataSource.java:400)
	at examples.databasetesting.EmployeeJDBC.create(EmployeeJDBC.java:75)
	at examples.databasetesting.EmployeeJDBCTest.create_theEmployeeIsPersisted(EmployeeJDBCTest.java:82)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:82)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:72)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:240)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:180)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:49)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
05-11 17:41:56 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Tabla "EMPLOYEE" no encontrada
Table "EMPLOYEE" not found; SQL statement:
delete from Employee where Id=? [42102-154]
05-12 05:52:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabla "EMPLOYEE" no encontrada
Table "EMPLOYEE" not found; SQL statement:
insert into PUBLIC.Employee (id, firstName, hireDate, lastName) values (null, ?, ?, ?) [42102-154]
05-12 05:52:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabla "EMPLOYEE" no encontrada
Table "EMPLOYEE" not found; SQL statement:
insert into PUBLIC.Employee (id, firstName, hireDate, lastName) values (null, ?, ?, ?) [42102-154]
05-12 05:58:51 jdbc[3]: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-154]
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:98)
	at org.h2.Driver.connect(Driver.java:72)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1148)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at org.springframework.orm.hibernate3.LocalDataSourceConnectionProvider.getConnection(LocalDataSourceConnectionProvider.java:81)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)
	at org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)
	at org.hibernate.jdbc.AbstractBatcher.prepareStatement(AbstractBatcher.java:116)
	at org.hibernate.id.IdentityGenerator$GetGeneratedKeysDelegate.prepare(IdentityGenerator.java:90)
	at org.hibernate.id.insert.AbstractReturningDelegate.performInsert(AbstractReturningDelegate.java:54)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:2176)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:2656)
	at org.hibernate.action.EntityIdentityInsertAction.execute(EntityIdentityInsertAction.java:71)
	at org.hibernate.engine.ActionQueue.execute(ActionQueue.java:279)
	at org.hibernate.event.def.AbstractSaveEventListener.performSaveOrReplicate(AbstractSaveEventListener.java:321)
	at org.hibernate.event.def.AbstractSaveEventListener.performSave(AbstractSaveEventListener.java:204)
	at org.hibernate.event.def.AbstractSaveEventListener.saveWithGeneratedId(AbstractSaveEventListener.java:130)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.saveWithGeneratedOrRequestedId(DefaultSaveOrUpdateEventListener.java:210)
	at org.hibernate.event.def.DefaultSaveEventListener.saveWithGeneratedOrRequestedId(DefaultSaveEventListener.java:56)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.entityIsTransient(DefaultSaveOrUpdateEventListener.java:195)
	at org.hibernate.event.def.DefaultSaveEventListener.performSaveOrUpdate(DefaultSaveEventListener.java:50)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.onSaveOrUpdate(DefaultSaveOrUpdateEventListener.java:93)
	at org.hibernate.impl.SessionImpl.fireSave(SessionImpl.java:563)
	at org.hibernate.impl.SessionImpl.save(SessionImpl.java:551)
	at org.hibernate.impl.SessionImpl.save(SessionImpl.java:547)
	at examples.databasetesting.hibernate.EmployeeHibernateTest.create(EmployeeHibernateTest.java:67)
	at examples.databasetesting.hibernate.EmployeeHibernateTest.delete_theEmployeeIsDeleted(EmployeeHibernateTest.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:74)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:82)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:72)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:240)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:180)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:49)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
